### DISCUSSION TITLE
KeeShare history

### DISCUSSION THREAD

**Discussion by LLKoder** [keepassxreboot/keepassxc#10119] [Q&A] [answered]
I'm interested how I can see KeeShare history. I mean what user/signer create a some new password, what user a some update password, etc.
I cannot open KeeShare password base directly.

**Comment by droidmonkey** [comment, marked as answer]
This isn't a feature of keeshare, or keepass databases. There is no  tracking of "users"

**Reply by LLKoder** [reply]
Do you mean KeePassXC or KeeShare have no this feature for now or something else? Can I do feature request?

**Reply by droidmonkey** [reply]
That isn't a feature that is likely to be implemented.  KeePass in general doesn't have a concept of an audit trail. Certainly not impossible to add a "last modified by" field, but anything beyond that would be going down the road of an audit log.

**Reply by LLKoder** [reply]
What you mean when say "audit log"? Is it log of KeePassXC or you mean something else? Where can I find it?
